packages/experimental/Drawer/android/.gitattributes

```diff
@@ -0,0 +1,6 @@
+#
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# These are explicitly windows files and should use crlf
```

Comment:
Interesting. I'm curious is this was needed because something broke? Asking because we have some similar android files in react-native-macos that maybe we should add this attribute to..

Reply:
hmm I'm not actually why this was needed. I'm not familiar with .gitattributes files. What is this supposed to do?

Reply:
Oh, I thought you added this file manually. Didn't realize it was autogenerated. Curious, how did you generate the template files for this native module? Might be good to update the [Contributing Guide](https://github.com/microsoft/fluentui-react-native/blob/master/CONTRIBUTING.md) with best practices afterwards.

https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings

Windows and macOS have different line endings, so git usually has to make a decision on which one to commit to source. I _think_ the standard has been to commit to source linux/macOS line endings (lf) and then if a git repo 
is checked out on windows, update the line endings to crlf on checkout (and back to lf when pushing back to source). I guess since .bat is a windows specific file, it's never meant to be run with linux//macOS line endings (lf), so this file tells git not to convert .bat files.

Reply:
https://docs.gradle.org/current/samples/sample_building_java_applications.html#run_the_init_task

I used the gradle init script to initialize the gradle project for Drawer. I can update the Contributing guide later! I might move the gradle project to a shared location in the components directory when we start creating native modules for other components as well